opds_springer/feed_generator.py

```diff
@@ -121,7 +121,7 @@ def opds_response_link(self, page_number, rel):
         """
         return {
             "rel": rel,
-            "href": f"https://ebooks-test.library.columbia.edu/static-feeds/springer/springer_feed_{page_number}.json",
+            "href": f"https://ebooks-test.library.columbia.edu/static-feeds/springer_test/springer_feed_{page_number}.json",
```

Comment:
I want to merge the other changes, but maybe the base url or a template string should go in the config?